tests/ext/unit/test_aiohttp.py

```diff
@@ -507,6 +508,9 @@ class UserSchema(marshmallow.Schema):
         )
 
         doc = hapic.generate_doc("aiohttp", "testing")
+        # Prevent keep of OrderedDict
```

Comment:
who is the author of the comment? what is the comment date?

Reply:
changed